### DISCUSSION TITLE
MDN roadmap for products and community?

### DISCUSSION THREAD

**Discussion by SphinxKnight** [mdn/mdn-community#418] [Announcements]
I perceived as ads and now "AI" as two "surprises" (and I gather from the discussions that I was not alone). Another topic is the infrequent/erratic "community handling" (rather than "community management) (despite the tone of the most recent article about this).

## Roadmap

1. Is there a public roadmap for MDN where such topics could have been seen?
2. What is the current "MDN product roadmap"? (incl. MDN Plus and *non* MDN Plus)

## Community

1. It seems that people managing/related to the community are not at Mozilla anymore: who is responsible for organizing/answering/channelling Mozilla's voice to the community?
2. Does Mozilla plan to invest more than just "damage control" in its community effort?

**Comment by caugner** [comment]
We have recorded your questions as numbers 31 (roadmap 1+2), 32 (responsibility) and 33 (community investment).

**Comment by SphinxKnight** [comment]
Addition: https://blog.mozilla.org/en/mozilla/mdn-web-documentation-collaboration/ does not contain any "AI" mention (as it was already the case). 

What is the purpose of this advertised roadmap?
Has Mozilla informed the quoted people in this article that the roadmap had been changed? or are they several roadmaps?

**Reply by SphinxKnight** [reply]
also see on https://github.com/orgs/mdn/discussions/335#discussioncomment-5289089 "Our focus is to continually make MDN better - that is why we have a fully scoped roadmap with pure MDN product improvements out there that we are committed to delivering by the end of the year. Ideally, this impacts our investment in our relationship with partners and the community. There are active conversations about improving our tooling and ways in which we support our partners and community."

**Comment by Rumyra** [comment]
Hi @SphinxKnight thanks for questioning & highlighting.

The public roadmap is for content only https://github.com/orgs/mdn/projects/26/views/7. There is no public roadmap for product, however we are looking into processes for contributors to give feedback product 👍

**Reply by SphinxKnight** [reply]
Thank you for the answer @Rumyra 🙇‍♂️
